### PR TITLE
Composite checkout: add delete buttons to review

### PR DESCRIPTION
**Pull request by fditrapani**
#### Changes proposed in this Pull Request

This PR adds a delete button to the WPcom review section. Since I tailored the review section so much for wpcom, I duplicated and baseline and added my code to a new wpcom specific file.

remove
 

#### Testing instructions

* Get checkout up and running with these instructions: https://github.com/Automattic/wp-calypso/pull/37013
* Proceed to the review step of checkout and remove a product from the cart.

### CODE REVIEW

File: packages/composite-checkout/src/wpcom/wp-checkout-order-review.js [hunk 1]
```diff
@@ -11,7 +11,12 @@ import styled from '@emotion/styled';
 import joinClasses from './join-classes';
 import Coupon from './coupon';
 import WPTermsAndConditions from './wp-terms-and-conditions';
-import { useLineItems, renderDisplayValueMarkdown, OrderReviewLineItems, OrderReviewTotal, OrderReviewSection } from '../public-api';
+import { useLineItems, renderDisplayValueMarkdown } from '../public-api';
+import {
+	OrderReviewLineItems,
```
Comment (sirbrillig) [inline packages/composite-checkout/src/wpcom/wp-checkout-order-review.js]: I like what you've done here, but can we rename these to start with `WP...`? I can see an argument for these simpler names, but until we move all these files to another place I'd worry about confusion where two components in one package that have the same name but different APIs.

Comment (fditrapani) [reply packages/composite-checkout/src/wpcom/wp-checkout-order-review.js]: Good call. I can change the names.

Comment (fditrapani) [reply packages/composite-checkout/src/wpcom/wp-checkout-order-review.js]: Updated!

File: packages/composite-checkout/src/components/checkout-modal.js [hunk 2]
```diff
@@ -150,7 +151,7 @@ function preventClose( event ) {
 
 function useModalScreen( isVisible, closeModal ) {
 	useEffect( () => {
-		document.body.style = isVisible ? 'overflow: hidden' : 'overflow: scroll';
+		document.body.style.cssText = isVisible ? 'overflow: hidden' : 'overflow: scroll';
```
Comment (fditrapani) [inline packages/composite-checkout/src/components/checkout-modal.js]: this fixed a really weird IE bug 🙄

**Pull request merged**